odata/types.go

```diff
@@ -40,6 +44,9 @@ type Type = string
 const (
 	TypeAccessPackage                               Type = "#microsoft.graph.accessPackage"
 	TypeAccessPackageAssignmentPolicy               Type = "#microsoft.graph.accessPackageAssignmentPolicy"
+	TypeAccessPackageQuestion                       Type = "#microsoft.graph.accessPackageQuestion"
+	TypeAccessPackageTextInputQuestion                       Type = "#microsoft.graph.accessPackageTextInputQuestion"
+	TypeAccessPackageMultipleChoiceQuestion                       Type = "#microsoft.graph.accessPackageMultipleChoiceQuestion"
```

Comment:
Not sure why the linter didn't pick this up, but could we tidy up the spacing for these?